src/cbang/time/Timer.h

```diff
@@ -37,10 +37,12 @@
 #undef min
 #endif // _WIN32
 
-
+#if defined(__GLIBC__)
```

Comment:
This also works when unconditionally (i.e. also for glibc) replacing the local declarations with a `#include <sys/time.h>`